tempodb/encoding/vparquet3/schema.go

```diff
@@ -213,6 +213,12 @@ type ResourceSpans struct {
 	ScopeSpans []ScopeSpans `parquet:"ss,list"`
 }
 
+type ServiceMeta struct {
+	ServiceName    string `parquet:",snappy,dict"`
+	SpanCount      uint64 `parquet:",delta"`
+	ContainsErrors bool   `parquet:""`
```

Comment:
We could just as easily record the number of errors instead of a single bool.  Any reason not to?